src/README.md

```diff
@@ -9,3 +9,53 @@ Key features include:
 - **Native integration with Pyroscope**: Seamlessly integrates with Pyroscope backend to provide detailed performance profiling and analysis.
 - **Query-Less Browsing**: Navigate profiling data without the need for complex queries.
 - **AI-Powered flame graph analysis**: uses a large-language model (LLM) to assist with flame graph data interpretation so you can identify bottlenecks, and get to the bottom of performance issues faster.
+
+> [!IMPORTANT]
+> Explore Traces is presently undergoing active development and is offered in a preview state. Subsequent updates are likely to incorporate significant changes that may impact existing functionality.
```

Comment:
```suggestion
> Explore Profiles is presently undergoing active development and is offered in a preview state. Subsequent updates are likely to incorporate significant changes that may impact existing functionality.
```

Reply:
@knylander-grafana Is this going to be rendered on GitHub? If not, we shouldn't use the

```
> [!IMPORTANT]
> ...
```

formatting as it's specific to GitHub markdown and not generally a markdown feature.

Reply:
Good point, Bryan. I'll update Mark's suggestion to include just a standard note that we use other places: 

>**Note**: